tests/restarting/from_5.0.0_until_6.3.0/CycleTestRestart-2.txt

```diff
@@ -1,5 +1,6 @@
 testTitle=Clogged
 runSetup=false
+storageEngineExcludeTypes = [4, 5]
```

Comment:
For 6.3 and before, the spec doesn't allow `[ ]`, you should use this syntax instead:
```
storageEngineExcludeTypes = 4,5
```